python/ray/dashboard/client/src/pages/dashboard/Dashboard.tsx

```diff
@@ -13,7 +13,7 @@ import LogicalView from "./logical-view/LogicalView";
 import NodeInfo from "./node-info/NodeInfo";
 import RayConfig from "./ray-config/RayConfig";
 import { dashboardActions } from "./state";
-import Tune from "./Tune";
+import TuneHolder from "./tune/TuneHolder";
```

Comment:
Aha I had a question about why did TuneHolder strike again but it turns out we actually need to for tab switching.

Reply:
@richardliaw do you want Tensorboard to exclusively show inside Tune tab? Maybe we should just have a separate tab so potentially we can render RLLib as well?